modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3870,6 +3882,23 @@ private void _validateGroupId(long groupId, String scope)
 		}
 	}
 
+	private void _validateListTypeEntryKey(
+			Map.Entry<String, Serializable> entry, String listTypeEntryKey,
+			ObjectField objectField)
+		throws PortalException {
+
+		ListTypeEntry listTypeEntry =
+			_listTypeEntryLocalService.fetchListTypeEntry(
+				objectField.getListTypeDefinitionId(), listTypeEntryKey);
+
+		if ((listTypeEntry == null) &&
+			(Validator.isNotNull(listTypeEntryKey) ||
+			 objectField.isRequired())) {
```

Comment:
@devpedrotavares Aren't you "reverting" the fix for [LPS-181610](https://issues.liferay.com/browse/LPS-181610) made in this [commit](https://github.com/liferay-objects/liferay-portal/commit/c9f555176e09fae80b3c5b291c4494e4b1b62b64) with the `Validator.isNotNull(listTypeEntryKey)?`

Reply:
Hey, @carolmariaabb. I talked to Marcela about this. I we stay without this check, we'll allow invalid picklist keys when the objectField it not required. Also, adding this back won't cause a regression on the mentioned ticket.